src/index.js

```diff
@@ -41,12 +42,52 @@ export const RCComponent = ({
   );
 
   useEffect(() => {
-    const cookiesPresent = Cookies.get('rc_token') && Cookies.get('rc_uid');
+    const cookiesPresent =
+      Cookies.get(RC_LOCAL_USER_TOKEN) && Cookies.get(RC_LOCAL_USER_ID);
     if (cookiesPresent) {
       setIsUserAuthenticated(true);
     }
   }, []);
 
+  const authenticatedUserUsername = useUserStore((state) => state.username);
+  const authenticatedUserAvatarUrl = useUserStore((state) => state.avatarUrl);
+  const authenticatedUserId = useUserStore((state) => state.userId);
+
+  const setAuthenticatedUserUsername = useUserStore(
+    (state) => state.setUsername
+  );
+  const setAuthenticatedUserAvatarUrl = useUserStore(
+    (state) => state.setUserAvatarUrl
+  );
+  const setAuthenticatedUserId = useUserStore((state) => state.setUserId);
+
+  useEffect(() => {
+    async function getUserEssentials() {
+      const res = await RCInstance.me();
+      setAuthenticatedUserAvatarUrl(res.avatarUrl);
+      setAuthenticatedUserUsername(res.username);
+      setAuthenticatedUserId(res.userId);
+    }
+
+    const cookiesPresent =
+      Cookies.get(RC_LOCAL_USER_TOKEN) && Cookies.get(RC_LOCAL_USER_ID);
+    if (cookiesPresent) {
+      setIsUserAuthenticated(true);
+    }
+
+    const storedUserId = localStorage.getItem('userId');
```

Comment:
change it to `rc_uid` -> RC_LOCAL_USER_ID so you don't use wrong keys

Reply:
Done